ledger-core/virtual/object/object.go

```diff
@@ -434,3 +435,31 @@ func (sm *SMObject) migrate(ctx smachine.MigrationContext) smachine.StateUpdate
 		return ctx.ReplaceWith(&smf)
 	})
 }
+
+type pendingCountersWarnMsg struct {
+	*log.Msg     `txt:"Pending counter does not match active records count in table"`
+	CounterType  string
+	PendingCount uint8
+	CountActive  uint8
+}
+
+func (sm *SMObject) correctionPendingCounters(logger smachine.Logger) {
+	unorderedCountActive := uint8(sm.PendingTable.GetList(contract.CallIntolerable).CountActive())
+	if sm.ActiveUnorderedPendingCount != unorderedCountActive {
```

Comment:
warning should be logged only if `sm.PendingTable.GetList(contract.CallIntolerable).Count()` != sm.ActiveUnorderedPendingCount. Because some of pendings could be finished in this pulse.
sm.ActiveUnorderedPendingCount  should always be set to sm.PendingTable.GetList(contract.CallIntolerable).CountActive()

I'd call this function as countActivePendings

Reply:
done